yellowbrick/model_selection/CVScores.py

```diff
@@ -0,0 +1,174 @@
+            for more information on the possible strategies that can be used here.
+
+    scoring : string, callable or None, optional, default: None
+            A string or scorer callable object / function with signature
+            ``scorer(estimator, X, y)``. 
+    
+            See scikit-learn model evaluation documentation for names of possible metrics.
+
+    kwargs : dict
+            Keyword arguments that are passed to the base class and may influence
+            the visualization as defined in other Visualizers.    
+                 
+    Examples
+    --------
+    
+    >>> from sklearn.model_selection import KFold, cross_val_score, 
+    >>> ShuffleSplit, StratifiedKFold
+    >>> from sklearn import datasets, svm, linear_model 
+
+    >>> iris = datasets.load_iris()
+    >>> clf = svm.SVC(kernel='linear', C=1)
+
+    >>> X = iris.data
+    >>> y = iris.target
+
+    >>> visualizer = CVScores(model=clf, cv=5, scoring='f1_macro')
+    >>> visualizer.fit(X,y)
+    >>> visualizer.poof()
+    
+    Notes
+    -----
+    
+    This visualizer is a wrapper around for the ``sklearn.model_selection.cross_val_score``
+    <<http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_val_score.html>>
+    
+    Refer to the scikit-learn
+    `cross-validation guide <http://scikit-learn.org/stable/modules/cross_validation.html>`
+    for more details
+    
+    """
+
+    def __init__(self, model, ax=None, cv=None, scoring=None, **kwargs):    
+                
+        super(CVScores, self).__init__(model, ax=ax, **kwargs)
+    
+        self.cv = cv
+        self.scoring = scoring
+
+    def fit(self, X, y, **kwargs):
+        """
+        Fits the learning curve with the wrapped model to the specified data.
+        Draws training and test score curves and saves the scores to the
+        estimator.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+                Training vector, where n_samples is the number of samples and
+                n_features is the number of features.
+
+        y : array-like, shape (n_samples) or (n_samples, n_features), optional
+                Target relative to X for classification or regression;
+                None for unsupervised learning.
+
+        Returns
+        -------
+        self : instance
+        
+        """
+
+        self.cv_scores_ = cross_val_score(self.estimator, X, y, cv=self.cv, scoring=self.scoring)
+        self.cv_scores_mean_ = np.mean(self.cv_scores_)
+
+        self.draw()
+        return self
+
+    def draw(self, **kwargs):
+        """
+        creates the bar chart of the CV scores generated from the fit method and places 
+        a dashed horizontal line that represents the average value of the CV scores
+        """
+        minimum = min(self.cv_scores_)
+        if minimum > 0.05:
+            minimum = minimum - 0.05 
+        plt.ylim(minimum, 1)
```

Comment:
Would you mind changing this to `self.ax.set_ylim(minium, 1)`? -- using the axes object on the visualizer ensures that weirdness doesn't occur. Also, if you remove this reference to `plt` you might also have to remove the import at the top to get the tests to pass.